_includes/footer.html

```diff
@@ -6,8 +6,13 @@
       <div class="grid-container">
         <div class="grid-row grid-gap">
           <div class="usa-footer__contact-links mobile-lg:grid-col-12">
-            <div class="usa-footer grid-row grid-gap-1">
-              <div class="grid-col-auto text-base-darkest">
+            <div class="usa-footer grid-row">
+              <div class="grid-col-auto">
+                This project is maintained by <a class="usa-link" href="https://18f.gsa.gov/">18F</a>
```

Comment:
Minor thing, I think you could pull in this link from the identifier data but either way works